codecov.yml

```diff
@@ -20,3 +20,6 @@ codecov:
   notify:
     require_ci_to_pass: false
     after_n_builds: 1  # send notifications and complete cc build after 1 upload
+
+ignore:
+  - "cognite/transformations_cli/commands/deploy/load_yaml.py"
```

Comment:
this is copied from cognite-extractor-utils, they already have tests for it in their repo. But if we touch it and break it,  we should be able to catch that bug before it is merged. Because config parsing tests should cover this.